tlscanary/template/js/report_page.js

```diff
@@ -8,7 +8,7 @@ function makeHeaderText(meta) {
   const desc = "Fx " + meta.test_metadata.app_version + " " + meta.test_metadata.branch 
           + " vs Fx " + meta.base_metadata.app_version + " " + meta.base_metadata.branch;
   const time = meta.run_start_time.split(".")[0].replace("T","-").replace(":","-").replace(":","-");
-  window.document.getElementById("header").innerHTML = "<h3 class=\"header\">" + desc + "<br>" + time + "</h3>";
+  window.document.getElementById("header").innerHTML = "<h3 class='header'>" + desc + "<br>" + time + "</h3>";
```

Comment:
Oh, are hard quotes even allowed in html? It definitely looks very alien to me. ```'<h3 class="header">'```, perhaps?